sdks/go/pkg/beam/model/pipeline_v1/beam_runner_api.pb.go

```diff
@@ -924,7 +924,7 @@ type PTransform struct {
 	//
 	Outputs map[string]string `protobuf:"bytes,4,rep,name=outputs" json:"outputs,omitempty" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
 	// (Optional) Static display data for this PTransform application. If
-	// there is none, or it is not relevant (such as use by the Fn API)
```

Comment:
Is this from a new go generate or a search'n'replace? If the latter, we should make the change to the proto file or revert.